eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -98,6 +100,10 @@ object Channel {
   case object PeriodicRefresh extends BroadcastReason
   case object Reconnected extends BroadcastReason
   case object AboveReserve extends BroadcastReason
+
+  sealed trait BitcoinEvent extends PossiblyHarmful
+  case object BITCOIN_FUNDING_PUBLISH_FAILED extends BitcoinEvent
+  case object BITCOIN_FUNDING_TIMEOUT extends BitcoinEvent
```

Comment:
Nit (I would still keep the `PossiblyHarmful` because `BITCOIN_FUNDING_PUBLISH_FAILED` is very sensitive): 

```suggestion
  private sealed trait BitcoinEvent extends PossiblyHarmful
  private final case object BITCOIN_FUNDING_PUBLISH_FAILED extends BitcoinEvent
  private final case object BITCOIN_FUNDING_TIMEOUT extends BitcoinEvent
```

There are other types in the companion object that should be made `final` and/or `private`. We should be a little more strict regarding visibility going forward, especially if we move to akka typed.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1759/commits/331147ff6b18f0c4500aed6d081f11ff6f647efa, we can fix the visibility of other fields when we modify them or when we move to akka-typed.